code-challenges.test.js

```diff
@@ -13,44 +13,122 @@
 
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
 
-// a) Create a test with expect statements for each of the variables provided.
-
 // Example input: 6
 // Expected output: [1, 1, 2, 3, 5, 8]
 
 // Example input: 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+// a) Create a test with expect statements for each of the variables provided.
+
+// a describe method that list the name of the function or naming of the particulat test
+describe("fibNum", () => {
+    it("the sequence in which each number is the sum of the two preceding ", () => {
+        expect(fibonacci(6)).toEqual(1, 1, 2, 3, 5, 8)
+    })
+})
+
```

Comment:
Nice job on the test. I see just one thing missing. The output you are expecting is an array. So you want to be sure to include the [] around your series of numbers. `.toEqual([1, 1, 2, 3, 5, 8])`